elide-core/src/main/java/com/yahoo/elide/parsers/state/RecordTerminalState.java

```diff
@@ -85,12 +84,8 @@ public RecordTerminalState(PersistentResource record, CollectionTerminalState co
 
     @Override
     public Supplier<Pair<Integer, JsonNode>> handleDelete(StateContext state) {
-        try {
-            record.deleteResource();
-            return () -> Pair.of(HttpStatus.SC_NO_CONTENT, null);
-        } catch (ForbiddenAccessException e) {
-            return () -> Pair.of(e.getStatus(), null);
-        }
+        record.deleteResource();
```

Comment:
I'm guessing that we'll have to be cleverer in `deleteResource` to throw `403` instead of `400`